pkg/operator/operator.go

```diff
@@ -69,6 +70,9 @@ func NewOpenShiftControllerManagerOperator(
 	// we only watch some namespaces
 	kubeInformersForOpenshiftControllerManager.Core().V1().Namespaces().Informer().AddEventHandler(c.namespaceEventHandler())
 
+	// set this bit so the library-go code employed for clusteroperator reconciliation allows this operator to be set to unmanaged
+	management.SetOperatorAlwaysManaged()
```

Comment:
wait..which direction are you trying to go?

Reply:
Calling this means you will *not* be able to set the operator to "unmanaged".

```
// SetOperatorAlwaysManaged is one time choice when an operator want to opt-out from supporting the "unmanaged" state.
// This is a case of control plane operators or operators that are required to always run otherwise the cluster will
// get into unstable state or critical components will stop working.
func SetOperatorAlwaysManaged() {
	allowOperatorUnmanagedState = false
}
```

Reply:
Yeah perhaps see my comment https://github.com/openshift/cluster-openshift-controller-manager-operator/pull/116#discussion_r317266003 perhaps

we need this call if we want to allow the operator to process getting set to unmanaged/removed, otherwise the library-go code will set the cluster operator conditions to UNKNOWN

if we decide the OCM operator should always stay in managed, then we remove this call and change the case statement starting at https://github.com/openshift/cluster-openshift-controller-manager-operator/pull/116/files#diff-554de5523753fda8c93e7008c9bd947fR84 to reject / ignore such a spec change.

Reply:
Based on the godoc comment I say we want to do this for ocm-o. Without it most of what makes OpenShift "OpenShift" breaks.

Reply:
OK @bparees and I had a side confab.  The net is:
a) we are going to employ `SetOperatorAlwaysManaged()`
b) we are not going to accept unmanaged/removed on the ocm object any longer